cmd/query.go

```diff
@@ -983,25 +983,29 @@ $ %s query unrelayed-pkts demo-path channel-0`,
 
 			src, dst := path.Src.ChainID, path.Dst.ChainID
 
-			c, err := a.config.Chains.Gets(src, dst)
+			chains, err := a.config.Chains.Gets(src, dst)
 			if err != nil {
 				return err
 			}
 
-			if err = c[src].SetPath(path.Src); err != nil {
-				return err
+			if chain, ok := chains[src]; ok {
+				if err = chain.SetPath(path.Src); err != nil {
```

Comment:
rather than nesting this, the check should check for `!ok` and return an error. same for L996